src/__test__/redux/actions/projects/createProject.test.js

```diff
@@ -1,51 +1,67 @@
 import configureStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
 import createProject from '../../../../redux/actions/projects/createProject';
-import initialState from '../../../../redux/reducers/projects';
-import { saveProject } from '../../../../redux/actions/projects';
-import { createExperiment } from '../../../../redux/actions/experiments';
-import { PROJECTS_CREATE } from '../../../../redux/actionTypes/projects';
+import { PROJECTS_CREATE, PROJECTS_SAVING, PROJECTS_SAVED } from '../../../../redux/actionTypes/projects';
 
-const mockStore = configureStore([thunk]);
-
-jest.mock('../../../../redux/actions/projects/saveProject');
-saveProject.mockImplementation(() => async () => { });
+import createExperiment from '../../../../redux/actions/experiments/createExperiment';
 
 jest.mock('../../../../redux/actions/experiments/createExperiment');
-createExperiment.mockImplementation(() => async () => ({
-  name: 'New project',
-  uuid: 'new-project',
-}));
+
+jest.mock('localforage');
+
+enableFetchMocks();
+
+const mockStore = configureStore([thunk]);
 
 describe('createProject action', () => {
```

Comment:
Maybe add test to check for error response?

Reply:
Added